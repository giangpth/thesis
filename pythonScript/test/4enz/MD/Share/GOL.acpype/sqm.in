Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C1        8.0010       90.9540       18.3910 
   8    O1        8.2070       89.5720       18.1930 
   6    C2        7.0520       91.5040       17.3380 
   8    O2        7.5310       91.2910       16.0200 
   6    C3        6.8000       92.9810       17.6310 
   8    O3        7.7150       93.8430       17.0120 
   1   H11        7.5720       91.1150       19.3800 
   1   H12        8.9560       91.4740       18.3230 
   1    H2        6.1030       90.9780       17.4370 
   1   H31        5.7990       93.2350       17.2830 
   1   H32        6.8460       93.1370       18.7090 
   1   HO1        8.8070       89.2380       18.8640 
   1   HO2        8.3800       91.7270       15.9130 
   1   HO3        7.5010       94.7520       17.2360 

