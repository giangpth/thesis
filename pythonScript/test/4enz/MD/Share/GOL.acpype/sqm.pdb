ATOM      1  C1  GOL  1113       8.082  91.051  18.580  1.00  0.00           C
ATOM      2  O1  GOL  1113       8.342  89.665  18.517  1.00  0.00           O
ATOM      3  C2  GOL  1113       7.041  91.466  17.536  1.00  0.00           C
ATOM      4  O2  GOL  1113       7.482  91.092  16.243  1.00  0.00           O
ATOM      5  C3  GOL  1113       6.754  92.966  17.614  1.00  0.00           C
ATOM      6  O3  GOL  1113       7.938  93.634  17.206  1.00  0.00           O
ATOM      7  H11 GOL  1113       7.700  91.234  19.618  1.00  0.00           H
ATOM      8  H12 GOL  1113       9.031  91.629  18.416  1.00  0.00           H
ATOM      9  H2  GOL  1113       6.087  90.887  17.686  1.00  0.00           H
ATOM     10  H31 GOL  1113       5.906  93.228  16.927  1.00  0.00           H
ATOM     11  H32 GOL  1113       6.492  93.258  18.665  1.00  0.00           H
ATOM     12  HO1 GOL  1113       8.482  89.444  17.586  1.00  0.00           H
ATOM     13  HO2 GOL  1113       8.216  91.677  16.011  1.00  0.00           H
ATOM     14  HO3 GOL  1113       7.718  94.569  17.125  1.00  0.00           H
