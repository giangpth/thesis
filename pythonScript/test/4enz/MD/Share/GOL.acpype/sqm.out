            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 19

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       32
| QMMM:    diag iterations used for timing =       20
| QMMM:
| QMMM:              Internal diag routine = 0.004918 seconds
| QMMM:                 Dspev diag routine = 0.008679 seconds
| QMMM:                Dspevd diag routine = 0.007921 seconds
| QMMM:                Dspevx diag routine = 0.100343 seconds
| QMMM:                 Dsyev diag routine = 0.043448 seconds
| QMMM:                Dsyevd diag routine = 0.051328 seconds
| QMMM:                Dsyevr diag routine = 0.018354 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.011969 seconds
| QMMM: Pseudo diagonalization appears to be slower than regular
| QMMM: diagonalization. Setting pseudo_diag=0 for optimum performance.
| QMMM:
| QMMM: Using internal diagonalization routine (diag_routine=1).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        8.0010   90.9540   18.3910
  QMMM:     2        2      O        8.2070   89.5720   18.1930
  QMMM:     3        3      C        7.0520   91.5040   17.3380
  QMMM:     4        4      O        7.5310   91.2910   16.0200
  QMMM:     5        5      C        6.8000   92.9810   17.6310
  QMMM:     6        6      O        7.7150   93.8430   17.0120
  QMMM:     7        7      H        7.5720   91.1150   19.3800
  QMMM:     8        8      H        8.9560   91.4740   18.3230
  QMMM:     9        9      H        6.1030   90.9780   17.4370
  QMMM:    10       10      H        5.7990   93.2350   17.2830
  QMMM:    11       11      H        6.8460   93.1370   18.7090
  QMMM:    12       12      H        8.8070   89.2380   18.8640
  QMMM:    13       13      H        8.3800   91.7270   15.9130
  QMMM:    14       14      H        7.5010   94.7520   17.2360

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -158.6464 kcal/mol        0.2629 kcal/(mol*A)
xmin    20     -158.6271 kcal/mol        1.5230 kcal/(mol*A)
xmin    30     -158.9231 kcal/mol        0.6423 kcal/(mol*A)
xmin    40     -160.1452 kcal/mol        2.8937 kcal/(mol*A)
xmin    50     -161.8630 kcal/mol        1.1272 kcal/(mol*A)
xmin    60     -162.7330 kcal/mol        0.4009 kcal/(mol*A)
xmin    70     -162.7624 kcal/mol        0.0232 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -40.1473  -37.8318  -36.5652  -31.1086  -26.3660  -22.6396  -19.3915  -18.2382
 -17.4918  -16.6786  -15.9201  -15.2784  -14.9480  -13.7540  -13.1767  -12.6168
 -11.6584  -11.4285  -10.6706    2.9377    3.2664    3.3761    3.5568    3.7782
   3.9586    3.9775    4.2072    4.6085    4.7190    5.7232    6.1428    6.4396


 Heat of formation   =       -162.76259234 kcal/mol  (       -7.05791563 eV)

 Total SCF energy    =     -33600.54178518 kcal/mol  (    -1457.02882725 eV)
 Electronic energy   =    -118140.76013187 kcal/mol  (    -5122.96778682 eV)
 Core-core repulsion =      84540.21834669 kcal/mol  (     3665.93895957 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.048
     2      O                 -0.326
     3      C                 -0.010
     4      O                 -0.329
     5      C                 -0.053
     6      O                 -0.340
     7      H                  0.106
     8      H                  0.080
     9      H                  0.127
    10      H                  0.079
    11      H                  0.076
    12      H                  0.211
    13      H                  0.217
    14      H                  0.210
 Total Mulliken Charge =      -0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -1.064    2.922   -0.002    3.110
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C        8.0823   91.0510   18.5796
  QMMM:     2        2      O        8.3421   89.6647   18.5171
  QMMM:     3        3      C        7.0406   91.4664   17.5363
  QMMM:     4        4      O        7.4818   91.0917   16.2433
  QMMM:     5        5      C        6.7538   92.9664   17.6143
  QMMM:     6        6      O        7.9380   93.6344   17.2063
  QMMM:     7        7      H        7.7002   91.2338   19.6176
  QMMM:     8        8      H        9.0312   91.6294   18.4156
  QMMM:     9        9      H        6.0865   90.8873   17.6864
  QMMM:    10       10      H        5.9062   93.2281   16.9271
  QMMM:    11       11      H        6.4919   93.2581   18.6652
  QMMM:    12       12      H        8.4819   89.4436   17.5857
  QMMM:    13       13      H        8.2158   91.6769   16.0109
  QMMM:    14       14      H        7.7176   94.5691   17.1245

           --------- Calculation Completed ----------

